sound/soc/intel/boards/sof_sdw.c

```diff
@@ -201,6 +201,16 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_RT715_DAI_ID_FIX |
 					SOF_SDW_PCH_DMIC),
 	},
+	{
+                .callback = sof_sdw_quirk_cb,
+                .matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Google"),
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Google_Brya0"),
+                },
+                .driver_data = (void *)(SOF_SDW_TGL_HDMI |
+                                       SOF_SDW_PCH_DMIC |
+                                       SOF_SDW_FOUR_SPK),
+        },
```

Comment:
@yongzhi1 Should we add the BT right away, or as separate patch?

Reply:
> @yongzhi1 Should we add the BT right away, or as separate patch?

To add BT quirks, can we make the changes here and https://github.com/thesofproject/linux/pull/2902 both places?

Reply:
Yes, @yongzhi1 we can add for both.